_posts/en/newsletters/2024-06-28-newsletter.md

```diff
@@ -0,0 +1,122 @@
+---
+title: 'Bitcoin Optech Newsletter #309'
+permalink: /en/newsletters/2024/06/28/
+name: 2024-06-28-newsletter
+slug: 2024-06-28-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Estimating the likelihood that an LN payment is feasible:** René
+  Pickhardt [posted][pickhardt feasible1] to Delving Bitcoin about
+  estimating the likelihood that an LN payment is feasible given the
+  public knowledge of a channel's maximum capacity but no knowledge
+  about its current balance distribution.  For example, Alice has a
+  channel with Bob and Bob has a channel with Carol.  Alice knows the
+  capacity of the Bob-Carol channel but she doesn't know how much of
+  that balance is controlled by Bob and how much is controlled by Carol.
+
+  Pickhardt notes that some wealth distributions are impossible in a
+  payment network.  For example, Carol can't receive more money in her
+  channel with Bob than the capacity of that channel.  When all the
+  impossible distributions are excluded, it can be useful to consider
+  all the remaining wealth distributions as equally likely to occur.
+  That can be used to produce a metric for the likelihood that a payment
+  is feasible.
+
+  For example, if Alice wants to send a 1 BTC payment to Carol, and the
+  only channels it can pass through are Alice-Bob and Bob-Carol, then
+  we can look at what percentages of wealth distributions in the
+  Alice-Bob channel and the Bob-Carol channel would allow that payment
+  to succeed.  If the Alice-Bob channel has capacity of several BTC,
+  most possible wealth distributions would allow the payment to succeed.
+  If the Bob-Carol channel has a capacity just barely over 1 BTC, then
+  most possible wealth distributions would prevent the payment from
+  succeeding.  This can be used to calculate the overall likelihood
+  of the feasibility of a payment of 1 BTC from Alice to Carol.
+
+  The likelihood of feasibility makes it clear that many LN payments
+  that naively seem possible will not succeed in practice.  It also
+  provides a useful basis for comparison: Pickhardt provides a graph for
+  a simulated network that compares the likelihood of feasibility to the
+  likelihood of a minimum cost flow solver making a successful payment,
```

Comment:
Just to clarify: I only compare the likelihood of feasibility with the likelihood that an **attempt** that is taken from the MCF approach is successful. The later makes no statement about the likelihood weather the **payment** will be successful or feasible.